.vscode/cspell.json

```diff
@@ -17,14 +17,15 @@
     ".vscode/tasks.json",
     "NOTICE.txt",
     "eng/",
-    "*.dict.txt",
+    "**/.dict.txt",
```

Comment:
@LarryOsterman I thought you checked this change in a while ago. Did it not get merged yet?

Reply:
It should have been merged with my changes on Friday.

Reply:
https://github.com/Azure/azure-sdk-for-rust/commit/11ae4ec2876f4a3bba50e71e3cd46bbcfa1c39d1